composer/trainer/ddp.py

```diff
@@ -42,6 +46,16 @@ class DDPSyncStrategy(StringEnum):
 
 @contextmanager
 def ddp_sync_context(state: State, is_final_microbatch: bool, sync_strategy: Union[str, DDPSyncStrategy]):
```

Comment:
Thoughts about making it private? It's used only by the trainer in one place.

Reply:
Yup sounds good!